docs/streams.html

```diff
@@ -87,6 +88,15 @@
 are leveraged by time-dependent operations such as joins.
 </p>
 
+<p>
+  Finally, whenever a Kafka Streams application writes records to Kafka, then it will also assign timestamps to these new records. The way the timestamps are assigned depends on the context:
+  <ul>
+    <li> When new output records are generated via processing some input record, for example, <code>context.forward()</code> triggered in the <code>process()</code> function call, output record timestamps are inherited from input record timestamps directly.</li>
+    <li> When new output records are generated via periodic functions such as <code>punctuate()</code>, the output record timestamp is defined as the current internal time (obtained through <code>context.timestamp()</code>) of the stream task.</li>
+    <li> For aggregations, the timestamp of a resulting aggregate update record will be that of the latest input record that triggered the update.</li>
```

Comment:
> will be that of the latest input record that triggered the update

I think we should clarify whether "latest" means "latest arrived" or "latest = largest timestamp".